docs/development/README.md

```diff
@@ -10,19 +10,17 @@ maintain this document updated. So, any new/modified action has to be written he
 - [`custom-ca`](custom-ca/): If you need to configure Istio to run with a specific CA
```

Comment:
Should this file `docs/development/README.md` moved inside istio katalog as `MAINTENANCE.md` ?

Reply:
🚀

Reply:
done ✅